### PR TITLE
Ui: Prevent toolSelector-scrollbar from looking ugly

### PR DESCRIPTION
**Pull request by chsterz**
StyleSheet needed to be enhanced with a selector
(QWidget#toolSelector{...}) in order to prevent inheritance.
This apparently affected the Scrollbar.
gamma_why

### CODE REVIEW

File: ui/mainwindow.ui [hunk 1]
```diff
@@ -89,7 +91,7 @@ background-position:bottom right;
      <x>0</x>
      <y>0</y>
      <width>829</width>
-     <height>27</height>
+     <height>39</height>
```
Comment (milianw) [inline ui/mainwindow.ui]: unrelated?

Comment (chsterz) [inline ui/mainwindow.ui]: I used the designer to change the stylesheet.
Was wondering myself about this.
So what I did was resetting the file and redo the changes to the stylesheet.
(I carefully looked not to accidentally move something in the designer) 
But again, this change co-ocurred after changing the style, so I thought if the designer thinks this is relevant to the change, I just commit that too.

Comment (milianw) [inline ui/mainwindow.ui]: Designer really likes to mess with dimensions when you edit something unrelated. Personally, I usually discard these using git after the fact. But I'm not forcing you to do it, just wanted to point it out - you already have my :+1:  :)

**Comment by krf** [comment]
:+1:, merging
